pkg/broker/broker.go

```diff
@@ -47,27 +47,27 @@ type serviceInstance struct {
 
 type broker struct {
 	// rwMutex controls concurrent R and RW access
-	rwMutex sync.RWMutex
+	rwMutex	    sync.RWMutex
 	// instanceMap maps instanceIDs to the ID's userProvidedServiceInstance values
-	instanceMap map[string]*serviceInstance
+	instanceMap map[string]*s3ServiceInstance
 	// client used to access s3 API
-	s3Client *minio.Client
+	s3Client    *minio.Client
 	// s3 server ip and port
-	s3url string // includes ":port"
-	s3ID  string
-	s3Pwd string
+	s3url	    string // includes ":port"
+	s3ID	    string
+	s3Pass	    string
 	// client used to access kubernetes
-	kubeClient *clientset.Clientset
+	kubeClient  *clientset.Clientset
 }
 
-// CreateBroker initializes the service broker. This function is called by server.Start()
```

Comment:
nit, we can take it or leave it: beginning package and function comments is the idiomatic way of documentation in Go: https://golang.org/doc/effective_go.html#commentary